### DISCUSSION TITLE
No more sugestion after paying subscription

### DISCUSSION THREAD

**Discussion by esQmo** [community/community#39883] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I have this weird bug. It just happened 2 days ego. After paying the subscription, Copilot won't do suggestions anymore. Disabling it, deleting etc didn"t fix the issue. What can I do?

No suggestions for any language.
Copilot on VS Code.

**Comment by davecheney** [comment]
Can you please paste the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.

**Reply by esQmo** [reply]
Here you are

`[INFO] [ghostText] [2022-11-24T07:35:16.430Z] Cancelled after requestMultiline
[INFO] [default] [2022-11-24T07:35:16.621Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T07:35:17.081Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T07:35:17.362Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 280 ms
[INFO] [streamChoices] [2022-11-24T07:35:17.371Z] solution 0 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-6G13N42tvRHLg98C8Dl9sZN9b6D5K}] created: [{1669275317}]
[INFO] [ghostText] [2022-11-24T07:35:17.385Z] Filtered out solution matching next line`

**Reply by esQmo** [reply]
![image](https://user-images.githubusercontent.com/10319812/203722184-48227408-c37e-4b1e-8594-1d228513e04f.png)

Sometimes it keeps search for suggestions indefinitely...

**Reply by davecheney** [reply]
It looks like copilot is working, sometimes it may not have anything to suggest.

**Reply by esQmo** [reply]
Well, that's weird. Because those are simple scenario where it worked normally but not since lately. I've been using it since the beginning and it has always worked, showing suggestions every time

**Reply by RMANOV** [reply]
Same state!

**Reply by esQmo** [reply]
You don't  get suggestions?

**Reply by RMANOV** [reply]
l does not get ANY suggestions - since of an hour or 2!
My subscription date is 21.12.2022.
Copilot stopped working exactly 2 days after paying the bill.
Definitely, there is a temporal connection!

**Comment by RMANOV** [comment]
[INFO] [default] [2022-11-24T12:10:49.706Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:10:50.069Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 362 ms
[INFO] [default] [2022-11-24T12:10:52.802Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:10:52.914Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 112 ms
[INFO] [default] [2022-11-24T12:10:53.697Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:10:53.891Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 193 ms
[INFO] [default] [2022-11-24T12:10:54.490Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:10:54.588Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 98 ms
[INFO] [default] [2022-11-24T12:10:54.836Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:10:54.922Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 85 ms
[INFO] [default] [2022-11-24T12:10:56.195Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:10:56.351Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 155 ms
[INFO] [default] [2022-11-24T12:10:58.648Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:10:58.839Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 190 ms
[INFO] [default] [2022-11-24T12:11:00.042Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:11:00.269Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 227 ms
[INFO] [default] [2022-11-24T12:11:02.679Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:11:02.763Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 83 ms
[INFO] [ghostText] [2022-11-24T12:14:08.267Z] Cancelled after extractPrompt
[INFO] [default] [2022-11-24T12:14:25.744Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:14:26.051Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 306 ms
[INFO] [default] [2022-11-24T12:14:27.757Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:14:27.846Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 88 ms
[INFO] [default] [2022-11-24T12:14:28.692Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:14:28.948Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 255 ms
[INFO] [default] [2022-11-24T12:14:30.051Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:14:30.157Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 106 ms
[INFO] [default] [2022-11-24T12:14:30.564Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:14:30.659Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 95 ms
[INFO] [default] [2022-11-24T12:14:32.472Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:14:32.622Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 150 ms
[INFO] [default] [2022-11-24T12:14:35.180Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:14:35.442Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 261 ms
[INFO] [default] [2022-11-24T12:15:04.791Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:15:04.890Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 99 ms
[INFO] [default] [2022-11-24T12:15:05.262Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:15:05.699Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:15:05.805Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 105 ms
[INFO] [default] [2022-11-24T12:15:07.479Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:15:07.634Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 155 ms
[INFO] [default] [2022-11-24T12:15:10.737Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:15:10.966Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 229 ms
[INFO] [default] [2022-11-24T12:15:12.191Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:15:12.271Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 79 ms
[INFO] [default] [2022-11-24T12:15:15.142Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:15:15.228Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 86 ms
[INFO] [default] [2022-11-24T12:15:15.659Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:15:15.743Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 84 ms
[INFO] [default] [2022-11-24T12:15:16.125Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:15:16.221Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 96 ms
[INFO] [default] [2022-11-24T12:15:18.602Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:15:18.781Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 179 ms
[INFO] [default] [2022-11-24T12:15:27.002Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:15:27.089Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 86 ms
[INFO] [default] [2022-11-24T12:15:27.504Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [ghostText] [2022-11-24T12:15:27.886Z] Cancelled after extractPrompt
[INFO] [default] [2022-11-24T12:15:27.984Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:15:28.071Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 87 ms
[INFO] [ghostText] [2022-11-24T12:15:28.548Z] Cancelled after extractPrompt
[INFO] [default] [2022-11-24T12:15:28.642Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:15:28.718Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 75 ms
[INFO] [default] [2022-11-24T12:15:29.140Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:15:29.227Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 86 ms
[INFO] [ghostText] [2022-11-24T12:15:29.508Z] Cancelled after extractPrompt
[INFO] [default] [2022-11-24T12:15:29.883Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:15:29.974Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 91 ms
[INFO] [ghostText] [2022-11-24T12:15:30.999Z] Cancelled after extractPrompt
[INFO] [default] [2022-11-24T12:15:31.394Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:15:31.635Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 241 ms
[INFO] [default] [2022-11-24T12:15:34.838Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:15:34.927Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 89 ms
[INFO] [default] [2022-11-24T12:15:35.333Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:15:35.423Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 90 ms
[INFO] [default] [2022-11-24T12:15:37.408Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:15:37.494Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 86 ms
[INFO] [ghostText] [2022-11-24T12:15:42.285Z] Cancelled by contextual filter
[INFO] [default] [2022-11-24T12:15:43.600Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:15:43.791Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 190 ms
[INFO] [default] [2022-11-24T12:15:44.280Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:15:44.369Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 89 ms
[INFO] [default] [2022-11-24T12:15:44.844Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:15:44.949Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 105 ms
[INFO] [default] [2022-11-24T12:15:45.793Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:15:45.879Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 86 ms
[INFO] [default] [2022-11-24T12:15:46.633Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [ghostText] [2022-11-24T12:15:47.026Z] Cancelled after extractPrompt
[INFO] [default] [2022-11-24T12:15:47.127Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:15:47.207Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 79 ms
[INFO] [default] [2022-11-24T12:15:47.872Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:15:47.962Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 90 ms
[INFO] [default] [2022-11-24T12:15:48.430Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:15:48.530Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 100 ms
[INFO] [ghostText] [2022-11-24T12:15:48.933Z] Cancelled after extractPrompt
[INFO] [ghostText] [2022-11-24T12:15:49.656Z] Cancelled after extractPrompt
[INFO] [default] [2022-11-24T12:15:49.752Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:15:49.905Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 152 ms
[INFO] [default] [2022-11-24T12:15:52.086Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:15:52.355Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:15:52.438Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 83 ms
[INFO] [default] [2022-11-24T12:15:54.806Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:15:54.945Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 139 ms
[INFO] [default] [2022-11-24T12:15:57.769Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:15:58.097Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 328 ms
[INFO] [default] [2022-11-24T12:15:59.547Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:15:59.636Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 88 ms
[INFO] [default] [2022-11-24T12:16:01.150Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:16:01.226Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 75 ms
[INFO] [default] [2022-11-24T12:16:03.037Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:16:03.227Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 189 ms
[INFO] [default] [2022-11-24T12:16:04.670Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:16:04.749Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 79 ms
[INFO] [default] [2022-11-24T12:16:05.136Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:16:05.215Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 78 ms
[INFO] [default] [2022-11-24T12:16:10.050Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:16:10.180Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 129 ms
[INFO] [default] [2022-11-24T12:16:32.254Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:16:32.552Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:16:32.649Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 97 ms
[INFO] [default] [2022-11-24T12:16:37.603Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:16:37.746Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 143 ms
[INFO] [default] [2022-11-24T12:16:41.183Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:16:41.271Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 87 ms
[INFO] [default] [2022-11-24T12:16:46.888Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:16:47.024Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 136 ms
[INFO] [default] [2022-11-24T12:16:48.833Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:16:49.332Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-11-24T12:16:49.432Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 100 ms

**Comment by RMANOV** [comment]
GitHub Copilot Nightly - same state - does not provide any suggestions! 
*To use GitHub Copilot Nightly, you need to uninstall GitHub Copilot - the regular version!
